addons/website_blog/tests/test_menu.py

```diff
@@ -0,0 +1,35 @@
+import odoo.tests
+
+@odoo.tests.tagged('post_install', '-at_install')
+class TestWebsiteBlogMenu(odoo.tests.HttpCase):
+
+    def test_menu_active_element(self):
+        # Create two published blog posts
+        post1 = self.env['blog.post'].create({
+            'name': "Blog Post 1",
+            'content': "This is the content of Blog Post 1",
+            'website_published': True,
+        })
+        post2 = self.env['blog.post'].create({
+            'name': "Blog Post 2",
+            'content': "This is the content of Blog Post 2",
+            'website_published': True,
+        })
+
+        # Add a link to each blog post in the website menu
+        menu = self.env.ref('website.main_menu')
+        menu.write({
+            'child_id': [(0, 0, {
+                'name': post1.name,
+                'url': post1.website_url,
+                'parent_id': menu.id,
+                'sequence': 10,
+            }), (0, 0, {
+                'name': post2.name,
+                'url': post2.website_url,
+                'parent_id': menu.id,
+                'sequence': 20,
+            })]
+        })
```

Comment:
that's a weird way to write it, why not just
```
self.env['website.menu'].create({
        'name': post1.name,
        'url': post1.website_url,
        'parent_id': menu.id,
        'sequence': 10,
}, {
        'name': post2.name,
        'url': post2.website_url,
        'parent_id': menu.id,
        'sequence': 20,
})
```

Even if you kept the way you wrote, then remove `'parent_id': menu.id,`, it's useless, it's setting again the relationship the other way around

Also note that `.write()` to write on a single field is to avoid, just do `menu.child_id = [Command.xxx()]`